ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -591,6 +591,82 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted-operator-ztp,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I don't think that we should have 2 blocking ZTP jobs
Please change this job into `always_run: false` and `optionl: true`.